pom.xml

```diff
@@ -62,45 +62,45 @@
             <dependency>
                 <groupId>org.apache.geronimo.components</groupId>
                 <artifactId>geronimo-transaction</artifactId>
-                <version>3.1.6-SNAPSHOT</version>
+                <version>4.0.0-SNAPSHOT</version>
             </dependency>
             <dependency>
                 <groupId>org.apache.geronimo.components</groupId>
                 <artifactId>geronimo-connector</artifactId>
-                <version>3.1.6-SNAPSHOT</version>
+                <version>4.0.0-SNAPSHOT</version>
             </dependency>
 
             <!-- external -->
             <dependency>
```

Comment:
think the new major is the opportunity to ditch this required dep and migrate to jul for consistency with the other stacks and new API.

Reply:
I am fine to migrate the slf4j dep to jul.

Reply:
done